metashare/api/models.py

```diff
@@ -90,3 +94,32 @@ def valid_token_for(self):
         if all(self.token) and self.org_id:
             return self.org_id
         return None
+
+
+class Product(HashIdMixin):
+    name = models.CharField(max_length=50, unique=True)
+    repo_url = models.URLField(unique=True)
```

Comment:
@wlonk Do we really want `repo_url` here, or would it be easier to use `repo_owner` and `repo_name`?

If we do stick with url, we need a way to standardize it. Right now for it to work you have to enter in the exact form returned from GH, e.g. `https://api.github.com/repos/jgerigmeyer/jquery-loading-overlay`. I don't think we want to require admins to know that?

Reply:
One advantage to using a full URL is that it's more flexible in case we ever want to support repositories stored somewhere other than github.com

Reply:
Yeah, that makes sense to me. In that case I think we should:

1. ignore `.git` endings
2. convert the repos returned from GH from e.g. `https://api.github.com/repos/...` to `https://github.com/...`